pages/card/card.html

```diff
@@ -4,27 +4,37 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="./card.css">
+  
   <link rel="stylesheet" href="https://starui-component-library.netlify.app/index.css">
-
+  <link rel="stylesheet" href="./card.css">
   <title>Document</title>
 </head>
 <body>
   <div class="product-card">
     <div class="card-image-container">
-      <img class="responsive-img" src="https://media.istockphoto.com/vectors/running-shoe-icon-isolated-sneaker-symbol-vector-vector-id1272854651?b=1&k=20&m=1272854651&s=170667a&w=0&h=PQhBbduJv9sIWEY47eIf3NGybbLL8jNt57Cu-ne__Ac=">
+      <img class="responsive-img" src="../../assets/real-madrid-21-22-home-jersey.jpg">
+      <i class="far fa-heart fa-lg"></i>
+    </div>
+    <h4>Real Madrid Jersey</h4>
+    <h4>₹2000</h4>
+    <button class="btn">Add to cart</button>
+  </div>
+  <div class="product-card">
+    <div class="card-image-container">
+      <img class="responsive-img" src="../../assets/real-madrid-21-22-home-jersey.jpg">
       <i class="far fa-heart fa-lg"></i>
```

Comment:
Also while hovering on this icon you can make its color red

Reply:
Yes but only the border color of the heart changes to red.
To make the actual heart fill with red we need to add a new class name.